README.md

```diff
@@ -57,7 +57,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: Constant - two nested loops but each are finite with a known number.
```

Comment:
This is a true statement. But that would also be the case for several of the examples above. Each of these loops is actually O(n) and because they iterate over different arrays we say O(n * m)